include/gmsh.tcc

```diff
@@ -153,8 +180,8 @@ void GMSH<Tdim, Tvertices>::store_element_vertices() {
   const unsigned firstelement = elements_.begin()->first;
   const unsigned lastelement = elements_.rbegin()->first;
 
-  std::array<double, Tvertices> elementkeyvalues;
-  std::array<double, Tdim * Tvertices> verticesarray;
+  Eigen::VectorXd elementkeyvalues(Tvertices);
+  Eigen::VectorXd verticesarray(Tdim * Tvertices);
 
   //! Iterate through element_
   for (unsigned i = firstelement; i <= lastelement; ++i) {
```

Comment:
Why is it `<=` and can you not iterate over the `elements_` container as such using a range iterator?